src/mem/ruby/protocol/MOESI_AMD_Base-CorePair.sm

```diff
@@ -575,9 +581,12 @@ machine(MachineType:CorePair, "CP-like Core Coherence")
                   if (in_msg.Type == RubyRequestType:LD) {
                     trigger(Event:C1_Load_L1miss, in_msg.LineAddress,
                             cache_entry, tbe);
-                  } else {
+                  } else if ((in_msg.Type == RubyRequestType:ST) || (in_msg.Type == RubyRequestType:ATOMIC)) {
```

Comment:
What does this have to do with the SQC commit?

Reply:
I incorrectly added this file. These changes were in my stash and I applied it wrongly added it when making review changes. Removed this from the commit now.

These changes are from when I added Flush support for L1,L2. Since GPUFS uses KVM instead of the CorePair, I didn't get to test this. Haven't made a commit for these changes because of this reason